tests/testthat/test_tune_tuneParamsMultiCrit.R

```diff
@@ -158,4 +186,27 @@ test_that("tuneParamsMultiCrit with resample.fun", {
   res = tuneParamsMultiCrit(lrn, binaryclass.task, rdesc, par.set = ps,
     measures = list(tpr, fpr), control = ctrl, resample.fun = constant05Resample)
   expect_true(all(getOptPathY(res$opt.path) == 0.5))
+
+  # MBO
+  ctrl = makeTuneMultiCritControlMBO(2L, budget = 4L * length(ps$pars) + 1L, learner = "regr.lm")
+  res = tuneParamsMultiCrit(lrn, binaryclass.task, rdesc, par.set = ps,
+    measures = list(tpr, fpr), control = ctrl, resample.fun = constant05Resample)
+  expect_true(all(getOptPathY(res$opt.path) == 0.5))
+})
+
+test_that("check n.objectives for MBO multi crit", {
+  lrn =  makeLearner("classif.rpart")
+  rdesc = makeResampleDesc("Holdout")
+  ps = makeParamSet(
+    makeIntegerParam("minsplit", lower = 1, upper = 50)
+  )
+
+  expect_error(makeTuneMultiCritControlMBO(1L))
```

Comment:
I know the checks `expect_error` are all very unspecific here. Could you improve all of them by writing `expect_error(..., "part of expected error message"`)? That would be very nice. This way we can check if the error is actually the error we are expecting.